qiskit/utils/mitigation/circuits.py

```diff
@@ -45,11 +48,11 @@ def count_keys(num_qubits: int) -> List[str]:
     additional_msg="For code migration guidelines, visit https://qisk.it/qi_migration.",
 )
 def complete_meas_cal(
-    qubit_list: List[int] = None,
-    qr: Union[int, List["QuantumRegister"]] = None,
-    cr: Union[int, List["ClassicalRegister"]] = None,
+    qubit_list: Sequence[int] | None = None,
+    qr: int | list | None = None,
+    cr: int | list | None = None,
     circlabel: str = "",
-) -> Tuple[List["QuantumCircuit"], List[str]]:
+) -> tuple[list, list[str]]:
```

Comment:
Why is this not `list["QuantumCircuit"]`?

Reply:
Types inside quotes can be used for [forward references](https://peps.python.org/pep-0484/#forward-references), this is not the case. But you're correct that this should be fixed.